### PR TITLE
3 pages more options fixed

### PR DESCRIPTION
**Pull request by Nppatel97**
This PR addresses the fix for the bug "more options not working on certain break-point". Currently it is addressed on Agenda, My Profile and My Tutors pages. More fixes are on the way. Some design fixes are also made like notifications pane now shows inside the viewport in windowed browser.

### CODE REVIEW

**Comment by sdevalapurkar** [comment]
This PR addresses this: https://github.com/sdevalapurkar/schedulearn/issues/105

File: schedule_lessons/dashboard/static/dashboard/css/agenda.css [hunk 1]
```diff
@@ -281,15 +283,48 @@ html body {
...
+    display: none !important;
+  }
+
+  #searchBar { 
+    left: 0%;
+  }
+}
+
 @media(min-width: 576px) {
+
```
Comment (sdevalapurkar) [inline schedule_lessons/dashboard/static/dashboard/css/agenda.css]: nit: remove newline

File: schedule_lessons/dashboard/static/dashboard/js/agenda.js [hunk 2]
```diff
@@ -10,6 +10,10 @@ $(document).ready(function () {
     // $('#options_btn').toggleClass('addBtnAnim');
   });
 
+  $("#secondNavBtn").click(function(){
```
Comment (sdevalapurkar) [inline schedule_lessons/dashboard/static/dashboard/js/agenda.js]: nit/consistency note: I personally prefer single quotes in javascript code but depending on how we have done it everywhere else i am ok with double quotes as long as we are consistent :) 
also, just check this out for fun if you'd like: https://bytearcher.com/articles/single-or-double-quotes-strings-javascript/

File: schedule_lessons/dashboard/static/dashboard/js/relationships.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
 $(document).ready(function () {
   $('[data-toggle="tooltip"]').tooltip();
 
+  $("#secondNavBtn").click(function(){
+    $("#navbarSupportedContent").slideToggle("fast");
```
Comment (sdevalapurkar) [inline schedule_lessons/dashboard/static/dashboard/js/relationships.js]: question: what does passing in 'fast' do? I know it is the duration of the toggle right?

File: schedule_lessons/dashboard/static/dashboard/css/agenda.css (hunk 1 above)
Comment (Nppatel97) [reply schedule_lessons/dashboard/static/dashboard/css/agenda.css]: About the code quality: We need to do some major work on the code in all the css files. There are unwanted, duplicate and unnecessary classes, things are sort of hard to find and overall a little messy. I'll take a thorough look at the code and we'll open a new PR if necessary.

File: schedule_lessons/dashboard/static/dashboard/js/agenda.js (hunk 2 above)
Comment (Nppatel97) [reply schedule_lessons/dashboard/static/dashboard/js/agenda.js]: I'll make it consistent. Thanks for noticing

File: schedule_lessons/dashboard/static/dashboard/js/relationships.js (hunk 3 above)
Comment (Nppatel97) [reply schedule_lessons/dashboard/static/dashboard/js/relationships.js]: Yeah, it toggles the menu faster. It matches the speed of the Bootstrap navbar.

File: schedule_lessons/dashboard/static/dashboard/css/agenda.css (hunk 1 above)
Comment (sdevalapurkar) [reply schedule_lessons/dashboard/static/dashboard/css/agenda.css]: sounds great man! would really appreciate that

File: schedule_lessons/dashboard/static/dashboard/js/relationships.js (hunk 3 above)
Comment (sdevalapurkar) [reply schedule_lessons/dashboard/static/dashboard/js/relationships.js]: cool, good to know!

**Comment by Nppatel97** [comment]
Alright, I think I have corrected all the pages that had more options button. They should all work now. Also fixed the navbar on schedule_lesson page so now it is consistent with rest of UI. Some other minor adjustments to the notifications panel were also made for mobile versions.

File: schedule_lessons/dashboard/static/dashboard/js/my_profile.js [hunk 4]
```diff
@@ -89,6 +89,10 @@ $(document).ready(function () {
     $("#reviewTab").fadeOut(300);
   });
 
+  $("#secondNavBtn").click(function(){
```
Comment (sdevalapurkar) [inline schedule_lessons/dashboard/static/dashboard/js/my_profile.js]: lets use single quotes here and in line below

File: schedule_lessons/dashboard/static/dashboard/js/relationships.js [hunk 5]
```diff
@@ -1,6 +1,10 @@
 $(document).ready(function () {
   $('[data-toggle="tooltip"]').tooltip();
 
+  $("#secondNavBtn").click(function(){
```
Comment (sdevalapurkar) [inline schedule_lessons/dashboard/static/dashboard/js/relationships.js]: single quotes

Comment (Nppatel97) [reply schedule_lessons/dashboard/static/dashboard/js/relationships.js]: Done. Merging the PR

**Pull request merged**